core/src/test/scala/integration/kafka/server/KRaftClusterTest.scala

```diff
@@ -1072,6 +1072,49 @@ class KRaftClusterTest {
     }
   }
 
+  @Test
+  def testCreateClusterAndCreateTopicWithRemoteLogManagerInstantiation(): Unit = {
+    val cluster = new KafkaClusterTestKit.Builder(
+      new TestKitNodes.Builder().
+        setNumBrokerNodes(3).
+        setNumControllerNodes(3).build())
```

Comment:
nit: 1 broker + 1 controller should be enough for this test scenario, right?

Reply:
agree, we don't need. fixed